pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -215,6 +215,75 @@ type heartbeat struct {
 	timestamp time.Time
 }
 
+var tierIndexers = cache.Indexers{
```

Comment:
are these simply moved out as new var or there is a diff here too.. couldn't immediately recognize the diff

Reply:
No difference here, just moved out to reuse it in test code.